src/framework/audio/devtools/waveformmodel.cpp

```diff
@@ -0,0 +1,108 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2021 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include "waveformmodel.h"
+
+using namespace mu::audio;
+
+static const volume_dbfs_t MAX_DISPLAYED_DBFS = 0.f; // 100%
+static const volume_dbfs_t MIN_DISPLAYED_DBFS = -60.f; // 0%
+
+WaveFormModel::WaveFormModel(QObject* parent)
+    : QObject(parent)
+{
+    playback()->audioOutput()->masterSignalAmplitudeChanged().onReceive(this, [this](const audioch_t, const float amplitude) {
+        setCurrentSignalAmplitude(amplitude);
+    });
+
+    playback()->audioOutput()->masterVolumePressureChanged().onReceive(this, [this](const audioch_t, const volume_dbfs_t pressure) {
+        if (pressure < MIN_DISPLAYED_DBFS) {
+            setCurrentVolumePressure(MIN_DISPLAYED_DBFS);
+        } else if (pressure > MAX_DISPLAYED_DBFS) {
+            setCurrentVolumePressure(MIN_DISPLAYED_DBFS);
```

Comment:
Should this maybe be <code>setCurrentVolumePressure(**MAX**_DISPLAYED_DBFS);</code>?

Reply:
yep, I've also noticed this yesterday

It has been fixed in another branch